### PR TITLE
Added scripts to download, install, start, and stop neo4j server

### PR DESCRIPTION
**Pull request by zhenlineo**
-Have not put it with existing runtest script together

### CODE REVIEW

File: neoctl.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+def main():
+    global neo4j_home
+    if len(argv) <= 1:
+        print_help()
+        exit()
+    try:
+        opts, args = getopt(argv[1:], "h", ["start=", "stop="])
+    except getopt.GetoptError as err:
+        print str(err)
```
Comment (spacecowboy) [inline neoctl.py]: This line will fail on Python3, you need parenthesis

    print(str(err))

File: neoctl.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+def powershell(cmd):
+
+    cmd = ['powershell.exe',
+           '-ExecutionPolicy', 'RemoteSigned',
+           'Import-Module', neo4j_home+'/bin/Neo4j-Management.psd1;'] + cmd
+    p = Popen(cmd, stdin=PIPE, stdout=PIPE, stderr=PIPE)
+    out, err = p.communicate()
+    return_code = p.wait()
+    print out
```
Comment (spacecowboy) [inline neoctl.py]: These 3 lines will fail on Python3, add parenthesis

File: neoctl.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    print err
+    print return_code
+
+
+def print_help():
+    print 'Usage:   neoctl.py <cmd=arg>'
```
Comment (spacecowboy) [inline neoctl.py]: Will fail on Python3

File: neoget.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+    if archive_name.endswith('zip') or archive_name.endswith('gz'):
+        stdout.write("Unzipping %s...\n" % archive_name)
+        zip_ref = zipfile.ZipFile(archive_name, 'r')
+        zip_ref.extractall(".")
+        zip_ref.close()
+
+
+def print_help():
+    print 'Usage:   neoget.py <cmd> [arg]'
```
Comment (spacecowboy) [inline neoget.py]: Also needs parenthesis

**Comment by spacecowboy** [comment]
To get consistent printing behavior, you might want to consider using

    from __future__ import print_function

File: neoctl.py (hunk 1 above)
Comment (zhenlineo) [inline neoctl.py]: Thank for all your catch for python 3. I was wondering why Nigel always uses stdout instead of print.

File: neoget.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    urlretrieve(archive_url, archive_name)
+
+    if archive_name.endswith('zip') or archive_name.endswith('gz'):
+        stdout.write("Unzipping %s...\n" % archive_name)
+        zip_ref = ZipFile(archive_name, 'r')
+        zip_ref.extractall(".")
+        zip_ref.close()
+
+
+def print_help():
```
Comment (spacecowboy) [inline neoget.py]: A common python pattern is to use the module's (file's) docstring as the help string. And then
using the triple quotes to preserve newlines. Might look a bit nicer than one call to print for each line.

For example:

```python
#!/usr/bin/env python
# -*- coding: utf-8 -*-

'''
Usage:   neoget.py <cmd> [arg]
         -v neo4j-version: download this specific neo4j version
         -l download-url : download neo4j provided by this url
         -h              : show this help message

Example: neoget.py -v 2.3.1
         neoget.py -h
'''


def print_help():
    print(__doc__)


if __name__ == '__main__':
    print_help()
```

Comment (zhenlineo) [inline neoget.py]: wow!! thanks a lot

Comment (nigelsmall) [inline neoget.py]: Good pattern :)

File: neoget.py [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+DIST = "http://dist.neo4j.org"
+DEFAULT_URL = "http://alpha.neohq.net/dist/neo4j-enterprise-3.0.0-M01-NIGHTLY-unix.tar.gz"
+WIN_URL = "http://alpha.neohq.net/dist/neo4j-enterprise-3.0.0-M01-NIGHTLY-windows.zip"
+
+
+def main():
+    is_windows = (name == 'nt')
+    archive_url = WIN_URL if is_windows else DEFAULT_URL
+    archive_name = path.split(archive_url)[1]
```
Comment (nigelsmall) [inline neoget.py]: As this is a URL, use `urlparse` as well for maximum correctness:
```
from urlparse import urlparse
path.split(urlparse(archive_url).path)[-1]
```

**Comment by nigelsmall** [comment]
Closing this PR as these files are now moved to https://github.com/nigelsmall/neokit

**Pull request closed without merging**